Sources/Defaults/Defaults+Bridge.swift

```diff
@@ -406,14 +414,42 @@ extension Defaults {
 			guard let value = value else {
 				return nil
 			}
+			guard
+				let cgColor = value.cgColor,
+				let colorSpace = cgColor.colorSpace?.name as? String,
+				let components = cgColor.components
+			else {
+				return NativeColor.bridge.serialize(NativeColor(value))
+			}
 
-			return NativeColor.bridge.serialize(NativeColor(value))
+			return [colorSpace, components]
 		}
 
 		public func deserialize(_ object: Serializable?) -> Value? {
-			guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+			if let object = object as? NativeColor.Serializable {
+				guard let nativeColor = NativeColor.bridge.deserialize(object) else {
+					return nil
+				}
+
+				return Value(nativeColor)
+			}
+
+			guard
+				let object = object as? [Any],
+				let rawColorspace = object[0] as? String,
+				let colorspace = CGColorSpace(name: rawColorspace as CFString),
+				let components = object[1] as? [CGFloat],
+				let cgColor = CGColor(colorSpace: colorspace, components: components)
+			else {
+				return nil
+			}
+			#if os(macOS)
+			guard let nativeColor = NativeColor(cgColor: cgColor) else {
 				return nil
 			}
+			#else
+			let nativeColor = NativeColor(cgColor: cgColor)
+			#endif
 
 			return Value(nativeColor)
```

Comment:
```suggestion
			return Value(cgColor)
```

`Swift.Color` accepts CGColor directly.

Reply:
`ColorBridge` is available for macOS 11.0.
But `SwiftUI.Color:init(cgColor:)` is only available in macOS 12.0 or newer 😢 .